services/ui-src/src/forms/mcpar/mcpar.json

```diff
@@ -20,6 +20,15 @@
           "form": {
             "id": "apoc",
             "fields": [
+              {
+                "id": "test1",
+                "type": "number",
+                "props": {
+                  "label": "Ratio Label",
+                  "hint": "Ratio Hint.",
+                  "mask": "ratio"
+                }
+              },
```

Comment:
Just making this note so we remember to remove this before merge

Reply:
+1